zenoh/src/net/routing/interceptor/access_control.rs

```diff
@@ -66,7 +66,7 @@ pub(crate) fn acl_interceptor_factories(
                     enforcer: Arc::new(policy_enforcer),
                 }))
             }
-            Err(e) => tracing::error!("Access control inizialization error: {}", e),
+            Err(e) => tracing::error!("Access control not enabled due to: {}", e),
```

Comment:
Since a security feature that the user wished to enable did not go through correctly, in my opinion it seems dangerous to proceed. Shouldn't we stop execution?

```suggestion
            Err(e) => panic!("Access control not enabled due to: {}", e),
```

Reply:
I think the upper logic should decide what to do with the execution. The interceptor has the freedom to return an error here and it should do so. I wouldn't panic directly at this level.